Gemfile

```diff
@@ -38,6 +38,7 @@ end
 gem 'activerecord-session_store', '~> 0.1.1'
 gem 'rails-observers', '~> 0.1'
 gem 'protected_attributes', '~> 1.1.1'
+gem 'roadie-rails'
```

Comment:
This should have version pinning based on the update policy of the gem, and the minimum version that the functionality required is in.